chain/client/src/client_actor.rs

```diff
@@ -1490,15 +1490,43 @@ impl ClientActor {
         Ok(epoch_start_sync_hash)
     }
 
+    fn ensure_sync_actors_running(&mut self, ctx: &mut Context<ClientActor>, shards: &[ShardId]) {
+        // TODO: Right now we just lazily start them here and never stop them. It would make sense to
+        // start these state sync actors when we recognize that state sync is needed, and then have them
+        // exit when they're done applying their shards' state. Then if in the future we need state sync again,
+        // we would start them again.
```

Comment:
We can add the change in this PR. 
Instead of starting it [here](https://github.com/near/nearcore/blob/master/chain/client/src/client.rs#L284)
we should start it when we initially send sync message.